doc/source/examples/dem/rotating-drum/rotating-drum.rst

```diff
@@ -0,0 +1,181 @@
+        set insertion frequency            		 			= 150000
+        set insertion box minimum x            	 			= -0.175
+        set insertion box minimum y            	        	= -0.07
+        set insertion box minimum z            	        	= 0
+        set insertion box maximum x            	        	= 0.175
+        set insertion box maximum y           	 			= 0.07
+        set insertion box maximum z            	        	= 0.09
+        set insertion distance threshold					= 1.575
+        set insertion random number range					= 0.05
+        set insertion random number seed					= 19
+    end
+
+
+Lagrangian physical properties
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The particles (226080 particles) are monodispersed, their diameter and density are 0.003 m and 2500 kg/m3, respectively.
+
+.. code-block:: text
+
+    subsection lagrangian physical properties
+        set gx            		 						= 0.0
+        set gy            		 						= 0.0
+        set gz            		 						= -9.81
+        set number of particle types	                = 1
+            subsection particle type 0
+            set size distribution type					= uniform
+                set diameter            	 			= 0.003
+                set number              				= 226080
+                set density particles  	 				= 2500
+                set young modulus particles         	= 100000000
+                set poisson ratio particles          	= 0.24
+                set restitution coefficient particles	= 0.97
+                set friction coefficient particles      = 0.3
+                set rolling friction particles         	= 0.01
+        end
+        set young modulus wall            				= 100000000
+        set poisson ratio wall            				= 0.24
+        set restitution coefficient wall           		= 0.85
+        set friction coefficient wall         			= 0.35
+        set rolling friction wall         	      	  	= 0.01
+    end
+
+
+Model parameters
+~~~~~~~~~~~~~~~~~
+
+Load-balancing updates the distribution of the subdomains between the processes in parallel simulation to achieve better computational performance (less simulation time). Three load-balancing methods are available in Lethe-DEM: ``once``, ``frequent``, or ``dynamic``. Read `this article <https://www.mdpi.com/2227-9717/10/1/79>`_ for more information about different load-balancing methods and their performances in various types of DEM simulations.
+
+Selecting ``repartition method = once``, requires defining the step at which the code calls load balancing (``load balance step``). ``Dynamic`` ``repartition method`` requires defining ``load balance frequency``, and in ``dynamic`` ``repartition method``, we should define ``load balance threshold`` and ``dynamic load balance check frequency``. In ``dynamic`` load balancing, the code checks the distribution of particles among the processors, every ``dynamic load balance check frequency`` steps, and if
```

Comment:
```suggestion
Selecting ``repartition method = once``, requires defining the step at which the code calls load balancing (``load balance step``). ``Frequent`` ``repartition method`` requires defining ``load balance frequency``, and in ``dynamic`` ``repartition method``, we should define ``load balance threshold`` and ``dynamic load balance check frequency``. In ``dynamic`` load balancing, the code checks the distribution of particles among the processors, every ``dynamic load balance check frequency`` steps, and if
```

Reply:
Isn't the method that uses load balance frequency called frequent and not dynamic?

Reply:
Absolutely right, I made a mistake here. Corrected.